lib/reporters/cli/index.js

```diff
@@ -26,7 +27,7 @@ var _ = require('lodash'),
         process: 'process',
         total: 'total'
     },
-    BODY_CLIP_SIZE = 2048,
+    DEFAULT_VERBOSE_OUTPUT_TRUNCATION_LIMIT = '2KB',
```

Comment:
Let's keep the constant name to body clip size. This way there is no ambiguity as to what output it is clipping.